### PR TITLE
some ARQ fixes and preparing for TCI

### PR DESCRIPTION
**Pull request by DJ2LS**
- introduce IRS buffer in NACK burst for comparing it with ISS buffer position
- if two stations connect at same time, as ISS we need to ignore the other ISS
- Tests with TCI
- Tests with response request 
- Tests with failing frame crc check

### CODE REVIEW

**Comment by DJ2LS** [comment]
@Mashintime i was surprisingly able fixing a minor arq protocol related bug.#325

File: tnc/rigctld.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def send_data_command(self, command, expect_answer) -> bytes:
                 while recv:
                     try:
 
-                        data = self.data_connection.recv(64)
+                        data = self.data_connection.recv(4800)
```
Comment (DJ2LS) [inline tnc/rigctld.py:185]: Not sure if this causes problems

**Comment by DJ2LS** [comment]
@Mashintime for some reason this is breaking png image preview
https://github.com/DJ2LS/FreeDATA/blob/ls-arq/gui/preload-chat.js#L446

File: tnc/rigctld.py (hunk 1 above)
Comment (Mashintime) [reply tnc/rigctld.py:185]: Seems to be working fine on my end.

**Comment by Mashintime** [comment]
> @Mashintime i was surprisingly able fixing a minor arq protocol related bug.#325 - at least I hope so

Nice job@DJ2LS !

**Comment by Mashintime** [comment]
> @Mashintime for some reason this is breaking png image preview https://github.com/DJ2LS/FreeDATA/blob/ls-arq/gui/preload-chat.js#L446

@DJ2LS should be resolved.

**Review by dk5sm — APPROVED** [review]
pretty much work. hope, this takes FD to another level of stability.

**Pull request merged**